crates/primitives/starknet/src/starknet_serde/mod.rs

```diff
@@ -102,6 +104,9 @@ pub enum DeserializeEventError {
     /// InvalidFromAddress error
     #[error(transparent)]
     InvalidFromAddress(#[from] Felt252WrapperError),
+    /// InvalidTransactionHash error
+    #[error("Invalid transaction hash format: {0}")]
+    InvalidTransactionHash(String),
```

Comment:
I couldn't use the `#[from]` macro here with `Felt252WrapperError`, decided I would just return the string representation of the invalid hash.

Reply:
why doesnt it work ?

Reply:
What prohibits using `#[from]` as in `InvalidFromAddress`?

Reply:
`#[from]` macro derives the `From<Felt252WrapperError>` implementation for `DeserializeEventError`. Since it's already derived for `InvalidFromAddress`, it cannot exist for `InvalidTransactionHash` as well, that would mean you have two implementations of `From<Felt252WrapperError>` for `DeserializeEventError`.

Reply:
Okay understood. Will leave for @EvolveArt to resolve if he doesn't have any further questions